docker-compose.yml

```diff
@@ -0,0 +1,23 @@
+version: '3.8'
+
+services:
+  jetfish-pg:
+    image: postgres:latest
+    volumes:
+      - postgres_data:/var/lib/postgresql/data
+    environment:
+      - POSTGRES_USER=postgres
+      - POSTGRES_PASSWORD=postgres
+  jetfish:
+    #image: ss14.jetfish
+    ports:
+      - "8050:80"
+    build:
+      context: ./
+    volumes:
+      - jetfish_data:/app/data/
```

Comment:
Comment out this whole block if this is invalid, which without an image I think it is

Reply:
The jetfish block is valid. It has a build sections

Reply:
We'll be building images of this through GitHub actions soon 🫡 
Then build can be replaced with image